docs/user/reference/metrics/string_list.md

```diff
@@ -8,252 +8,353 @@ _Note:_ Be careful using arbitrary strings and make sure they can't accidentally
 
 ---
 
-## Configuration
+## Recording API
 
-First you need to add an entry for the counter to the `metrics.yaml` file:
+### `add`
 
-```YAML
-search:
-  engines:
-    type: string_list
-    description: >
-      Records the name of the enabled search engines.
-    lifetime: application
-    ...
-```
-
-## API
+Add a new string to the list.
 
 {{#include ../../../shared/tab_header.md}}
-
 <div data-lang="Kotlin" class="tab">
 
 ```Kotlin
 import org.mozilla.yourApplication.GleanMetrics.Search
 
-// Add them one at a time
-engines.forEach {
-  Search.engines.add(it)
-}
+Search.engines.add("wikipedia")
+Search.engines.add("duck duck go")
+```
 
-// Set them in one go
-Search.engines.set(engines)
+</div>
+<div data-lang="Java" class="tab"></div>
```

Comment:
(this applies to all tab groups) Would you mind als adding Java examples? I am assuming Java provides this API since it is available in Kotlin. Follow-up would be fine too.

Reply:
Yeah, let me do that.